src/api/ledger/parse/cancellation.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+const assert = require('assert');
+
+function parseOrderCancellation(tx) {
+  assert(tx.TransactionType === 'OfferCancel');
+  return {
+    orderSequence: tx.OfferSequence
```

Comment:
Nice catch, `fee` isn't part of the `specification`, so I should probably move it to `outcome`.